.jenkins/cscs-ault/env-hipcc.sh

```diff
@@ -16,6 +16,7 @@ spack_spec="pika@main+rocm arch=${spack_arch} %${spack_compiler} malloc=system c
 
 configure_extra_options+=" -DCMAKE_BUILD_RPATH=$(spack location -i ${spack_compiler})/lib64"
 configure_extra_options+=" -DCMAKE_HIP_ARCHITECTURES=gfx900;gfx906"
+configure_extra_options+=" -DWHIP_BACKEND=HIP"
```

Comment:
This needs to be reverted before merging.

Reply:
Just to put the explanation here as well: I originally added `WHIP_BACKEND` here because `whip` was being fetched and configured by pika's CMake configuration. CI will use an externally installed `whip` before this is merged, and then `whip` will already be configured with either CUDA or HIP.